test/unit/ec2_udev_rules/test_ec2_dev_2_volid.py

```diff
@@ -31,3 +31,116 @@ def test_validate_device_name(device_name, raises):
         assert_that(validate_device_name).raises(ValueError).when_called_with(device_name).contains("invalid pattern")
     else:
         assert_that(validate_device_name(device_name)).is_true()
+
+
+@pytest.mark.parametrize(
+    ("status_code", "content", "expected_value"),
+    [(200, {"key": "value"}, {"X-aws-ec2-metadata-token": {"key": "value"}}), (400, {"key": "value"}, {})],
+)
+def test_get_imdsv2_token(mocker, status_code, content, expected_value):
+    mock = mocker.Mock()
+    mocker.patch("requests.put", mock)
+    mock.return_value.status_code = status_code
+    mock.return_value.content = content
+    assert_that(get_imdsv2_token()).is_equal_to(expected_value)
+
+
+@pytest.mark.parametrize(
+    ("dev", "expected_name", "raises"),
+    [
+        ("nvme0n1", "sdf", True),
+        ("nvme0n1p1", "sdf", True),
+        ("nvme0n1p128", "sdf", True),
+        ("xvd0n1", "/dev/sd0n1", False),
+        ("xvd0n1p1", "/dev/sd0n1p1", False),
+    ],
+)
+def test_convert_dev(mocker, dev, expected_name, raises, capsys):
```

Comment:
As you can see here we're duplicating the test code, because there is a code duplication. 
We have the same function defined in two different python modules.

We should instead:
1. rename manageVolume.py to manage_volume.py to have the same convention of ec2_dev_2_volid.py 
2. have a common module called for example ec2_volume.py with all the functions that are used by the two modules and import them to avoid duplication.

Anyway we can do it in a future patch.

Reply:
Makes sense, I went ahead and did those in separate commits.

In the most recent commit, I changed manageVolume.py to manage_volume.py.

In the commit before that, I added the ec2_volume.py file and took out the duplicate functions from the other two modules and put it in there. One question: for adapt_device_name (previously convert_dev), the two modules actually have slight differences for that function. I left them as-is in their own files, but are they supposed to be different?